lstchain/data/effective_time.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import astropy.units as u
+
+def get_effecive_time(events):
+    """
+    Calculate the effective observation time of a set 
+    of real data events.
+    Parameters
+    ----------
+    events: pandas DataFrame
+
+    Returns
+    -------
+    teff: float
+    """
+    
+    deltat = np.diff(events.dragon_time)
```

Comment:
You can just change `events.dragon_time` to 'events["dragon_time"].value' so that you can use the next lines of codes as is.